server/src/database/setup.ts

```diff
@@ -1,63 +1,70 @@
 import { readFile } from "fs";
 import parentLogger from "../../utils/logger";
 import db from "./db";
-import { PreReq, CoReq, CourseSection, Course } from "./schema";
+import { PreReq, CoReq, Course } from "./schema";
 
 const log = parentLogger.child({ module: "router" });
 const src = "./utils/output.json";
 
 export const setupDb = async () => {
     let { rows: tables } = await getTables();
     tables = tables.map((table) => table.table_name);
-    if (["prereq", "coreq", "course", "coursesection"].every((name) => tables.includes(name))) {
+    if (["prereq", "coreq", "course"].every((name) => tables.includes(name))) {
         log.info("Tables already exist. Will not be re-creating the DB.");
         return;
     }
     await dropDb();
     await createDb();
-    await popDb();
+    await populateDb();
 };
 
-const dropDb = () => db.query(`DROP TABLE IF EXISTS PreReq, CoReq, Course, CourseSection;`);
+const dropDb = () => db.query(`DROP TABLE IF EXISTS PreReq, CoReq, Course;`);
 
```

Comment:
Okay, correct me if I'm wrong, but it looks like we aren't deleting the CourseSection table yet - took a look at what's in my local db. I think it's because you've replaced `CourseSection` with `Course`. We should probably keep `CourseSection` here initially.

Reply:
nope that makes sense! added the coursesection